investment-app/TRINV.Api/TRINV.Application/Commands/Notifications/CreateNotificationCommand.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace TRINV.Application.Commands.Notifications;
+
+using Domain.Entities;
+using MediatR;
+using System.ComponentModel.DataAnnotations;
+using Domain.Validations;
+using Shared.Business.Utilities;
+using Shared.Business.Exceptions;
+using Enums;
+using Interfaces;
+using Shared.Business.Extension;
+
+public record CreateNotificationCommand : IRequest<OperationResult>
+{
+    [Required]
+    public int NotificationType { get; set; }
+
+    [Required]
+    [MaxLength(200)]
+    public string Message { get; set; } = string.Empty;
+}
+
+internal class CreateNotificationCommandHandler : IRequestHandler<CreateNotificationCommand, OperationResult>
+{
+    readonly IRepository<Notification> _repository;
+    readonly IUnitOfWork _unitOfWork;
+
+    public CreateNotificationCommandHandler(IUnitOfWork unitOfWork, IRepository<Notification> repository)
+    {
+        _unitOfWork = unitOfWork;
+        _repository = repository;
+    }
+
+    public async Task<OperationResult> Handle(CreateNotificationCommand request,
+        CancellationToken cancellationToken)
```

Comment:
why this is one the next line. Seems strange

Reply:
Could be ReSharper